From 1bbcdec78c5de4f4c0128a75fab1a5ab44ea1dd2 Mon Sep 17 00:00:00 2001
From: Eugene Syromiatnikov <esyr@redhat.com>
Date: Tue, 18 Feb 2025 16:54:03 +0100
Subject: [PATCH 12/17] releasenote.md: add information about 06-ba-08
 microcode file

This microcode file was first appeared as part of microcode-20240312,
was present in the following releases up to microcode-20241112,
and was removed in microcode-20250211.  While [1] mentions
that there are no products in production that would use that microcode,
it does not warrant creation of discrepancy between the release notes
and the actually shipped contents;  moreover, presence of 60BA8 CPUID
in [2] hints at its possible availability, after all.

NB: This CPUID can still be updated with the latest 06-ba-0* microcode
even despite removal of the microcode file, as it shares the microcode
data with 06-ba-02 and 06-ba-03 microcode files, that still contain
it in the extended signature:

    % iucode_tool -L microcode-20250211/intel-ucode/06-ba-02
    microcode bundle 1: microcode-20250211/intel-ucode/06-ba-02
      001/001: sig 0x000b06a2, pf_mask 0xe0, 2024-07-31, rev 0x4124, size 220160
               sig 0x000b06a2, pf_mask 0xe0, 2024-07-31, rev 0x4124
               sig 0x000b06a3, pf_mask 0xe0, 2024-07-31, rev 0x4124
               sig 0x000b06a8, pf_mask 0xe0, 2024-07-31, rev 0x4124

[1] https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/issues/83#issuecomment-2346609663
[2] https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01139.html

* releasenote.md (microcode-20240312): Mention addition of the microcode
at revision 0x4121.
(microcode-20240910): Mention an update to revision 0x4122.
(microcode-20241112)

Signed-off-by: Eugene Syromiatnikov <esyr@redhat.com>
---
 releasenote.md | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/releasenote.md b/releasenote.md
index 5184613..1e212cc 100644
--- a/releasenote.md
+++ b/releasenote.md
@@ -148,6 +148,7 @@
 |:---------------|:---------|:------------|:---------|:---------|:---------
 | SPR-SP         | E2       | 06-8f-05/87 | 2b000603 |          | Xeon Scalable Gen4
 | SPR-SP         | E3       | 06-8f-06/87 | 2b000603 |          | Xeon Scalable Gen4
+| RPL            |          | 06-ba-08/e0 | 00004123 |          | Core Gen13
 
 ### New Disclosures Updated in Prior Releases
 
@@ -190,6 +191,7 @@
 | EMR-SP         | A0       | 06-cf-01/87 | 21000230 | 21000283 | Xeon Scalable Gen5
 | EMR-SP         | A1       | 06-cf-02/87 | 21000230 | 21000283 | Xeon Scalable Gen5
 | MTL            | C0       | 06-aa-04/e6 | 0000001f | 00000020 | Core™ Ultra Processor
+| RPL            |          | 06-ba-08/e0 | 00004122 | 00004123 | Core Gen13
 | RPL-H/P/PX 6+8 | J0       | 06-ba-02/e0 | 00004122 | 00004123 | Core Gen13
 | RPL-HX/S       | C0       | 06-bf-02/07 | 00000036 | 00000037 | Core Gen13/Gen14
 | RPL-S          | H0       | 06-bf-05/07 | 00000036 | 00000037 | Core Gen13/Gen14
@@ -253,6 +255,7 @@
 | ADL            | R0       | 06-9a-04/80 | 00000433 | 00000434 | Core Gen12
 | ADL-N          | N0       | 06-be-00/11 | 00000017 | 0000001a | Core i3-N305/N300, N50/N97/N100/N200, Atom x7211E/x7213E/x7425E
 | MTL            | C0       | 06-aa-04/e6 | 0000001e | 0000001f | Core™ Ultra Processor
+| RPL            |          | 06-ba-08/e0 | 00004121 | 00004122 | Core Gen13
 | RPL-E/HX/S     | B0       | 06-b7-01/32 | 00000123 | 00000129 | Core Gen13/Gen14
 | RPL-H/P/PX 6+8 | J0       | 06-ba-02/e0 | 00004121 | 00004122 | Core Gen13
 | RPL-HX/S       | C0       | 06-bf-02/07 | 00000035 | 00000036 | Core Gen13/Gen14
@@ -438,6 +441,7 @@
 | EMR-SP         | A-1      | 06-cf-02/87 |          | 21000200 | Xeon Scalable Gen5
 | EMR-SP         | A-0      | 06-cf-01/87 |          | 21000200 | Xeon Scalable Gen5
 | MTL            | C-0      | 06-aa-04/e6 |          | 0000001c | Core™ Ultra Processor
+| RPL            |          | 06-ba-08/e0 |          | 00004121 | Core Gen13
 
 ### Updated Platforms
 
-- 
2.13.6

